lld/test/ELF/bp-section-orderer.s

```diff
@@ -0,0 +1,336 @@
+# REQUIRES: aarch64
+
+# RUN: rm -rf %t && split-file %s %t
+# RUN: llvm-mc -filetype=obj -triple=aarch64 %t/a.s -o %t/a.o
+# RUN: llvm-profdata merge %t/a.proftext -o %t/a.profdata
+
+# The code below is the reduced version of the output
```

Comment:
See https://llvm.org/docs/TestingGuide.html#elaborated-tests

you may want `-fno-exceptions -fno-rtti -fno-asynchronous-unwind-tables`